cg/services/deliver_files/file_formatter/utils/sample_service.py

```diff
@@ -28,8 +31,7 @@ def _create_sample_folders(ticket_dir_path: Path, sample_names: set[str]):
             sample_dir_path = Path(ticket_dir_path, sample_name)
             sample_dir_path.mkdir(exist_ok=True)
 
-    def _format_sample_files(self, sample_files: list[SampleFile]) -> list[FormattedFile]:
-        formatted_files: list[FormattedFile] = self._get_formatted_files(sample_files)
+    def _format_sample_files(self, formatted_files: list[FormattedFile]) -> list[FormattedFile]:
```

Comment:
This naming is a bit unclearer now - I think we are using the word "format" too much. Would `rename_sample_files` (or maybe `rename_files`) be better here?